mcs_benchmark_data/models/benchmark.py

```diff
@@ -14,4 +18,23 @@ class Benchmark(_Model):
     abstract: str
     author: Tuple[str, ...]
     dataset: Tuple[BenchmarkDataset, ...]
-    submissions: Tuple[Submission, ...]
\ No newline at end of file
+    submissions: Tuple[Submission, ...]
+
+    def to_rdf(
+        self, *, graph: Graph, **kwds) -> Resource:
+        resource = _Model.to_rdf(
+            self, graph=graph, **kwds
+        )
+        resource.add(RDF.type, MCS[self.__class__.__name__])
+
+        #How to add name? FOAF?

```

Comment:
Whatever is in the example outputs in the ontology docs. I believe Henrique's ontology uses schema.org across the board, so this is probably schema:name, schema:description, et al.